.gitignore

```diff
@@ -2,4 +2,4 @@
 /vendor
 composer.phar
 composer.lock
-.DS_Store
+.DS_Store
```

Comment:
This is macOS specific : https://en.wikipedia.org/wiki/.DS_Store
Deport it in your global `.gitingore`

```suggestion
```